_dashboards/admin-ui-index/index-management.md

```diff
@@ -0,0 +1,223 @@
+
+For example, you can define a policy that moves your index into a **read_only** state after 30 days and then deletes it after a set period of 90 days. You can also set up the policy to send you a notification message when the index is deleted.
+
+You might want to perform an index rollover after a certain amount of time or run a **force_merge** operation on an index during off-peak hours to improve search performance during peak hours.
+
+To apply a policy, select the index you want to apply the policy to in the **Indices** list under **Index Management**. Then select the **Actions** button, and select **Apply policiy** from the drop-down menu.
+
+<img src="{{site.url}}{{site.baseurl}}/images/admin-ui-index/apply-policy.PNG" alt="User interface showing apply policy prompt">
+
+#### Close
+
+The [close]({{site.url}}{{site.baseurl}}/api-reference/index-apis/close-index/) operation closes an index. Once an index is closed, you cannot add data to it or search for any data within the index.
+
+To close an index, select the index you want to close in the **Indices** list under **Index Management**. Then select the **Actions** button, and select **Close** from the drop-down menu.
+
+#### Open
+
+The [open]({{site.url}}{{site.baseurl}}/api-reference/index-apis/open-index/) index operation opens a closed index, letting you add or search for data within the index.
+
+To open an index, select the index you want to open in the **Indices** list under **Index Management**. Then select the **Actions** button, and select **Open** from the drop-down menu.
+
+#### Reindex
+
+The [reindex]({{site.url}}{{site.baseurl}}/api-reference/document-apis/reindex/) operation lets you copy all or a subset of your data from a source index into a destination index.
+
+To reindex an index, select the index you want to reindex in the **Indices** list under **Index Management**. Then select the **Actions** button, and select **Reindex** from the drop-down menu.
+
+<img src="{{site.url}}{{site.baseurl}}/images/admin-ui-index/reindex-expanded.png" alt="User interface showing reindex prompt">
+
+#### Shrink
+
+The [shrink]({{site.url}}{{site.baseurl}}/api-reference/index-apis/shrink-index/) index operation moves all of your data in an existing index into a new index with fewer primary shards.
+
+To shrink an index, select the index you want to shrink in the **Indices** list under **Index Management**. Then select the **Actions** button, and select **Shrink** from the drop-down menu.
+
+<img src="{{site.url}}{{site.baseurl}}/images/admin-ui-index/shrink.png" alt="User interface showing shrink prompt">
+
+#### Split
+
+The [split]({{site.url}}{{site.baseurl}}/api-reference/index-apis/split/) index operation splits an existing read-only index into a new index, cutting each primary shard into some amount of primary shards in the new index.
```

Comment:
```suggestion
The [split]({{site.url}}{{site.baseurl}}/api-reference/index-apis/split/) index operation splits an existing read-only index into a new index, splitting each primary shard into a number of primary shards in the new index.
```

Reply:
Change [split] to [split index] to following naming convention in the preceding sections.